site/public/js/server.js

```diff
@@ -221,6 +221,32 @@ function newUploadCourseMaterialsForm() {
 
 }
 
+function newEditCourseMaterialsForm(dir, this_file_section, this_hide_from_students) {
+
+    //let url = buildCourseUrl(["course_materials", "edit"]) + "?path=" + path;
+    //$('.popup-form').css('display', 'none');
+    console.log(this_hide_from_students);
+    var form = $("#edit-course-materials-form");
+    if(this_hide_from_students == "on"){
+        $("#hide-materials-checkbox", form).prop('checked',true);
+    }
+    else{
+        $("#hide-materials-checkbox", form).prop('checked',false);
+    }
+    if(this_file_section != null){
+        for(let index = 0; index < this_file_section.length; ++index){
+            console.log(this_file_section[index]);
+            $("#section-" + this_file_section[index], form).prop('checked',true);
+        }
+    }
+    $("#material-edit-form", form).attr('data-directory', dir);
+    form.css("display", "block");
+    //$('[name="edit-course-material-message"]', form).html('');
+    //$('[name="edit-course-material-message"]', form).append('<b>'+file_name+'</b>');
+    //$('[name="edit-confirmation"]', form).attr('action', url);
+    //form.css("display", "block");
```

Comment:
is the commented out lines here (and console.log) necessary to keep or can we remove them?

Reply:
We can remove them, the version I pushed was meant to get feedback on a specific issue and wasn't ready to be merged.